frontend/src/js/controllers/changePwdCtrl.js

```diff
@@ -7,67 +7,82 @@
         .module('evalai')
         .controller('ChangePwdCtrl', ChangePwdCtrl);
 
-    ChangePwdCtrl.$inject = ['utilities', 'validationSvc', '$state', '$http', '$rootScope'];
+    ChangePwdCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
 
-    function ChangePwdCtrl(utilities, validationSvc, $state, $http, $rootScope) {
+    function ChangePwdCtrl(utilities, $state, $http, $rootScope) {
         var vm = this;
         var userKey = utilities.getData('userKey');
         vm.wrnMsg = {};
         vm.isValid = {};
+        vm.user = {};
 
+        vm.startLoader = function(msg) {
+            $rootScope.isLoader = true;
+            $rootScope.loaderTitle = msg;
+            vm.loginContainer.addClass('low-screen');
```

Comment:
Do one thing too. Give name it `vm.changepassContainer` instead of `vm.loginContainer`.